drivers/hmis_external_apis/extensions/hmis/unit_type_extension.rb

```diff
@@ -16,6 +16,17 @@ module UnitTypeExtension
                 -> { where(namespace: HmisExternalApis::AcHmis::Mper::SYSTEM_ID) },
                 class_name: 'HmisExternalApis::ExternalId',
                 as: :source
+        has_many :external_unit_availability_syncs, class_name: 'HmisExternalApis::AcHmis::UnitAvailabilitySync', dependent: :destroy
+      end
+
+      # @param project_id [Integer] Hmis::Hud::Project.id
+      # @param user_id [Integer] Hmis::User.id
+      def track_availability(project_id:, user_id:)
+        HmisExternalApis::AcHmis::UnitAvailabilitySync.upsert_or_bump_version(
```

Comment:
Can we add a
`return unless mper_id.present?` ? we don't want to run this for non-MPER-linked unit types. Other installations may also use "unit types".